pvlib/test/test_tools.py

```diff
@@ -1,6 +1,10 @@
+import numpy as np
+import pandas as pd
 import pytest
 
+import pvlib.tools
```

Comment:
`tools` is already imported on the next line.

Reply:
Ähh right! ;D